playbooks/provision.yml

```diff
@@ -102,6 +102,11 @@
       become: yes
       tags: logrotate
 
+    - role: newrelic
+      become: yes
+      when: new_relic_api_key is defined
```

Comment:
This role also depends on `new_relic_account_id` and `new_relic_region`, so I was wondering if we should check for those here too. 
But then if you forgot to set any of those, it would just silently fail. 
👍 So I think this way is best: if you intended to switch it on, but left out a detail, then it would be better to get an error message which shows what's missing.